ADDONS.md

```diff
@@ -116,38 +117,35 @@ module RubyLsp
 
       sig do
         override.params(
+          response_builder: ResponseBuilders::Hover,
```

Comment:
We need to mention that not all requests have a dedicated response builder. Some use `ResponseBuilders::CollectionResponseBuilder[ResponseDataType]` instead.